### PR TITLE
fixes #23

### LINKED ISSUE #23: add new option for EditElement for set the value from clipboard
This option doesn't use `urpa.AppElement.send_text` method but add the value from clipboard.

### PR DESCRIPTION
**Pull request by KaterinaCZ**

### CODE REVIEW

File: urpaform/elements.py [hunk 1]
```diff
@@ -41,6 +43,8 @@ def __init__(
         value_is_in="value",
         clear_keys=("CTRL+A", "DEL"),
         default_value="",
+        clipboard=False,
```
Comment (Slavaqq) [inline urpaform/elements.py]: I would use more universal name for this attribute (maybe `setter_method` or `setter_type) with default value string. I assume we will add another method in the feature.

File: urpaform/elements.py [hunk 2]
```diff
@@ -41,6 +43,8 @@ def __init__(
         value_is_in="value",
         clear_keys=("CTRL+A", "DEL"),
         default_value="",
+        clipboard=False,
+        paste_keys="ALT+E+V"
```
[the remark this answers is not in the document]
Comment (KaterinaCZ) [reply urpaform/elements.py]: I do not know. Which shortcut would you mark as the most used one for automations with URPA tools? Checking it now, I am not even sure about the possibility to send just **anything** (any shortcut or whatever else) as value of that attribute. Should I add some limitation? Like:
```
_PASTE_KEYS_ARE_IN = ("ALT+E+V", "CTRL+C")
...
        paste_keys = paste_keys.upper()
        if paste_keys not in self._PASTE_KEYS_ARE_IN:
            raise ValueError(f"Value in argument paste_keys must be from: '{self._PASTE_KEYS_ARE_IN}'!")
        self.paste_keys = paste_keys
```

[the remark this answers is not in the document]
Comment (Slavaqq) [reply urpaform/elements.py]: I don´t think we want the limitation of the past in option. The keys very often depends on the aplication. So use as default `"CTRL+V"` a remove the limitation.

File: urpaform/elements.py [hunk 3]
```diff
@@ -95,7 +117,11 @@ def _clear(self):
 class PasswordElement(_FormElement):
     """A class used to represent a Passwordbox in a form."""
 
-    def __init__(self, element, show_in_log=False, clear_keys=("CTRL+A", "DEL")):
+    _PASTE_KEYS_ARE_IN = ("ALT+E+V", "CTRL+V")
+    _SEND_METHOD_IS_IN = ("send_text", "send_key")
+
+    def __init__(self, element, show_in_log=False, clear_keys=("CTRL+A", "DEL"), send_method="send_text",
+                 paste_keys="ALT+E+V"):
```
Comment (Slavaqq) [inline urpaform/elements.py]: default for `paste_keys` should be same as in EditElement

File: urpaform/elements.py [hunk 4]
```diff
@@ -31,7 +33,9 @@ def __str__(self):
 class EditElement(_FormElement):
     """A class used to represent a common Editbox in a form."""
 
+    _PASTE_KEYS_ARE_IN = ("ALT+E+V", "CTRL+V")
     _VALUE_IS_IN = ("value", "name", "text_value")
+    _SEND_METHOD_IS_IN = ("send_text", "send_key")
```
Comment (Slavaqq) [inline urpaform/elements.py]: I not sure about send_text but I don´t  like the name. I definitly know that send_key is wrong name for this option.

**Pull request merged**